service/src/routes/api.ts

```diff
@@ -0,0 +1,62 @@
+import express from 'express'
+import dotenv from 'dotenv'
+import axios, { AxiosInstance, AxiosResponse }from 'axios'
+
+export const register = (app: express.Application) => {
+
+    //Get a list of tasks from Mavenlink
+    app.get("/tasks", async (req, res) => {
+        const options = {
+            headers:{
+                Authorization: "Bearer " + process.env.MAVENLINK_TOKEN
+            }
+        };
+        axios.get(process.env.MAVENLINK_TASK_URL, options)
+        .then(response => {
+            console.log(response.data.stories);
+        })
+        .catch(error => console.log(error));
+    });
+
+    //Create a task in Mavenlink
+    app.get("/create", async (req, res) => {
```

Comment:
```suggestion
    app.get("/mavenlink/tasks/create", async (req, res) => {
```

Could we do something like this? That way we make things more explicit as in what belongs to mavenlink and what to us. Moreover, by doing something like this we don't have to change these URIs when we code the reports. What do you think?